security/login_link.rst

```diff
@@ -0,0 +1,510 @@
+    {
+        /**
+         * @Route("/login", name="login")
+         */
+        public function requestLoginLink(LoginLinkHandlerInterface $loginLinkHandler, UserRepository $userRepository, Request $request)
+        {
+            // check if login form is submitted
+            if ($request->isMethod('POST')) {
+                // load the user in some way (e.g. using the form input)
+                $email = $request->request->get('email');
+                $user = $userRepository->findOneBy(['email' => $email]);
+
+                // create a login link for $user
+                $loginLinkDetails = $loginLinkHandler->createLoginLink($user);
+                $loginLink = $loginLinkDetails->getUrl();
+
+                // ... send the $loginLink (e.g. using the Mailer or Notifier component)
+
+                // render a "Login link is sent!" page
+                return $this->render('security/login_link_sent.html.twig');
+            }
+
+            // if it's not submitted, render the "login" form
+            return $this->render('security/login.html.twig');
+        }
+    }
+
+.. code-block:: html+twig
+
+    {# templates/security/login.html.twig #}
+    {% extends 'base.html.twig' %}
+
+    {% block body %}
+    <form action="{{ path('login') }}" method="POST">
+        <input type="email" name="email">
+        <button type="submit">Login</button>
+    </form>
+    {% endblock %}
+
+In this controller, the user is submitting their e-mail address to the
+controller. Based on this property, the correct user is loaded and a login
+link is created using
+:method:`Symfony\\Component\\Security\\Http\\LoginLink\\LoginLinkHandlerInterface::createLoginLink`.
+
+.. caution::
+
+    It is important to send this link to the user and **not show it directly**,
+    as that would allow anyone to login. For instance, use the
+    :doc:`mailer </mailer>` component to mail the login link to the user.
+    Or use the :doc:`notifier </notifier>` component to send an SMS to the
+    user's device.
+
+.. note::
+
+    The service or controller using the ``LoginLinkHandlerInterface`` must
+    be used only if the current request is covered by a
+    :ref:`security firewall <security-firewalls>`.
+
+Import Considerations
+---------------------
+
+Login links are a convinient way of authenticating users, but it is also
+considered less secure than a traditional username and password form. It is
+not recommended to use login links in security critical applications.
+
+However, the implementation in Symfony does have a couple extension points
+to make the login links more secure. In this section, the most important
+configuration decisions are discussed:
+
+* `Limit Login Link Lifetime`_
+* `Invalidate Login Links`_
+* `Only allow a Link to be used Once`_
+
+Limit Login Link Lifetime
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is important for login links to have a limited lifetime. This reduces
+the chance that someone can intercept the link and use it to login as
```

Comment:
Maybe the word « chance » is not appropriate 
Maybe use « the possibility/the risk » instead ?